code/modules/mob/living/carbon/superior_animal/superior_animal.dm

```diff
@@ -196,7 +202,7 @@
 	var/ranged = FALSE  //Do we have a range based attack?
 	var/rapid = FALSE   //Do we shoot in groups?
 	var/rapid_fire_shooting_amount = 3 //By default will rapid fire in 3 shots per.
-	var/projectiletype  //What are we shooting?
+	var/obj/item/projectile/projectiletype  //What are we shooting?
```

Comment:
Why are you doing this?
Dosnt this limit to only being projectiles being exspected?

Reply:
i mean, we dont have aby mob that fires something that isnt a projectile

Reply:
plus i just was told that assigning a variable of a different type to a typed variable doesnt cause bad things as long as you dont try to access typed vars